### DISCUSSION TITLE
Can't Index the Originals Folder

### DISCUSSION THREAD

**Discussion by Thorium9O** [photoprism/photoprism#4643] [Q&A]
Howdy everyone,

I have a large folder of pictures I'm trying to set up in PhotoPrism. I would like to avoid importing them all and creating a copy in the storage folder, as there are a large number. I've tried setting up the folder containing the photos as the original folder and indexing it, but PhotoPrism is not finding anything. Is this the correct way to do it?

Thanks in advance,

**Comment by omerdduran** [comment]
Hi there. If you are using Docker Compose, you can set it like this example in the yml file. 

```yaml
volumes:
  - “/path/to/your/photos:/photoprism/originals”
```

Common issues if photos aren't being found:

Permissions - Container needs read access
Path mapping - Double check volume paths

If you have checked them and you still can't see the photos, you can check the logs. Also the storage folder and the originals folder are different things. The storage folder stores config, cache, backup, thumbnail and sidecar files. 

https://docs.photoprism.app/getting-started/docker-compose/#photoprismoriginals

**Reply by Thorium9O** [reply]
Hi Omer, 
I think that is what i have. I've included my conf file below. I double check the permissions and all users are rw in both docker and on the drive. I can see the originals dir and the Film_Drive subdir. 
Thanks for the help!


services:
  photoprism:
    image: photoprism/photoprism:latest
    container_name: photoprism
    restart: unless-stopped
    ports:
      - "----:2342"  
    environment:
      PHOTOPRISM_ADMIN_PASSWORD: "--------"
      PHOTOPRISM_STORAGE_PATH: "/photoprism/storage"
      PHOTOPRISM_IMPORT_PATH: "/photoprism/import"
      PHOTOPRISM_DEFAULT_TLS: "true"  
      PHOTOPRISM_DISABLE_CHOWN: "true"  
    volumes:
      - "/run/media/base/Seagate/photoprism:/photoprism/storage"
      - "/run/media/base/Seagate/photoprism/originals:/photoprism/originals 
      - "/run/media/base/Seagate/Film_Drive:/photoprism/originals/Film_Drive
      - "/run/media/base/Seagate/photoprism/import:/photoprism/import"
    user: "1000:1000"

**Reply by omerdduran** [reply]
Hello. I see that there are missing quotation marks in the volume mappings.  
```yaml
volumes:
  - "/run/media/base/Seagate/photoprism:/photoprism/storage"
  - "/run/media/base/Seagate/photoprism/originals:/photoprism/originals"  # Missing quote
  - "/run/media/base/Seagate/Film_Drive:/photoprism/originals/Film_Drive" # Missing quote
  - "/run/media/base/Seagate/photoprism/import:/photoprism/import"
```

also please check the following

https://docs.photoprism.app/getting-started/troubleshooting/docker/#overlay-volumes

**Reply by Thorium9O** [reply]
Hi Omer,
I double-checked the docker-compose.yml, and it had the quotation marks. It seems my post must have had a copy-paste error.

As for the link, I'm running this on Manjaro. I was under the impression that systemd handles the tasks that cgroupfs-mount would handle on Ubuntu. Is that not correct? When I look in Docker I can see the originals and the Film_Drive folders. Does that not mean that the linking of the directories worked?
Thanks again.
